torch/quantization/fx/qconfig_utils.py

```diff
@@ -172,7 +228,9 @@ def check_is_valid_qconfig_dict(qconfig_dict: Any) -> None:
       `qconfig_dict`: dictionary whose keys we want to check
     """
 
-    qconfig_dict_allowed_keys = {"", "object_type", "module_name_regex", "module_name"}
+    qconfig_dict_allowed_keys = {
+        "", "object_type", "module_name_regex", "module_name",
+        "module_name_object_type_order"}
```

Comment:
nit: should this just be `object_type_order`?

Reply:
hmm, it is `object_type_order` within the forward function of `module_name`.  Thoughts on if there is a better way to name it?